src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/adaptive/AdaptiveJqlMemory.kt

```diff
@@ -18,6 +19,16 @@ class AdaptiveJqlMemory(
 
     companion object {
         fun getAvailableTags(): List<String> = BuiltInJQL.values().map { it.name }.toList()
+
+        fun JqlMemory.simple() = filterJqls { tag ->
```

Comment:
Is it better than a simple class implemented with `: JqlMemory by TagSelectiveJqlMemory`?

Reply:
I didn't want to spill it beyond `AdaptiveJqlMemory`, where the `recallByTag` and `BuiltInJQL` originate from.